### PR TITLE
Frontend UI & API calls to few Spotify end-points

### PR DESCRIPTION
**Pull request by tejasraibagi26**
# Frontend UI

### Additions
- ## Home Page
  - ### Desktop View Screen Shot 2022-03-23 at 9 15 22 PM
  - ### Tablet View Screen Shot 2022-03-23 at 9 15 36 PM
  - Mobile View (Navbar not responsive yet) Screen Shot 2022-03-23 at 9 15 44 PM
- ## Login with Spotify
  - Currently won't work (requires server side callback)

### CODE REVIEW

**Comment by troytomasch** [comment]
A couple notes
- Could we add a little more padding between the scroll bars and the text above them?
- If the window is too short then I'm unable to go down to the categories scroll bar
- For the messages sidebar -
- I'm not sure if "Search chat" is the correct wording. Does it search for messages or for chat rooms themselves?
- When I hover over the icon for a chat room, the text changes from gray to white, unless that has different functionality from clicking on the rest of the button, I don't think it should change colors

File: src/Components/Messages/Messages.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    },
+    {
+      id: Math.round(Math.random() * 10000000),
+      title: "Electro",
+    },
+    {
+      id: Math.round(Math.random() * 10000000),
+      title: "Chill",
+    },
+  ];
```
Comment (troytomasch) [inline src/Components/Messages/Messages.jsx:23]: Will this be populated with the message rooms from the database?

**Review by troytomasch — CHANGES_REQUESTED** [review]
I left a comment with a few minor cosmetic changes

File: src/Components/Messages/Messages.jsx (hunk 1 above)
Comment (tejasraibagi26) [reply src/Components/Messages/Messages.jsx:23]: Yes, these are only created as placeholders.

**Comment by tejasraibagi26** [comment]
> A couple notes
> 
> * Could we add a little more padding between the scroll bars and the text above them?
> * If the window is too short then I'm unable to go down to the categories scroll bar
> * For the messages sidebar -
> * I'm not sure if "Search chat" is the correct wording. Does it search for messages or for chat rooms themselves?
> * When I hover over the icon for a chat room, the text changes from gray to white, unless that has different functionality from clicking on the rest of the button, I don't think it should change colors

- Changed placeholder from "Search chat..." to "Search chatroom..."
- Added padding between text and scrollbars
- Chatroom hover icon bug fixed
- Layout not yet perfectly responsive for mobile view.

**Comment by Sach24eth** [comment]
All good from my end. Do you want to have the chat room navigation on the same "homepage" as such instead of a separate page?

**Comment by tejasraibagi26** [comment]
> All good from my end. Do you want to have the chat room navigation on the same "homepage" as such instead of a separate page?

It's better to have the chat on the same layout, so the user can chat and browse at the same time.

**Pull request merged**